Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{HOMEBREW_CACHE}/bazel-workdir"
```

Comment:
No matter what the answer is, we don't expect formulae to write directly to the cache and we don't have code that would be capable of cleaning up arbitrary directories in the Homebrew cache. Thus this will need to be changed to something else. IMO, this should be always somewhere in `buildpath` where it will be cleaned up once the build concludes.